godef.go

```diff
@@ -36,20 +37,45 @@ func fail(s string, a ...interface{}) {
 }
 
 func init() {
-	// take GOPATH, set types.GoPath to it if it's not empty.
-	p := os.Getenv("GOPATH")
-	if p == "" {
-		return
+	if os.Getenv("GO15VENDOREXPERIMENT") != "" {
+		setupVendorExperiment()
```

Comment:
You should at least catch the case where it's `== "0"` so that it can be turned off.

Quoting the [design document](https://docs.google.com/document/d/1Bz5-UB7g2uPBdOx-rw5t9MxJwkfpx90cqG9AFL0JAYo/edit):

> Projects containing “vendor” directories could still use “GO15VENDOREXPERIMENT=0” to get the old behavior while they convert their code. In a still later release (possibly Go 1.7) we would remove the use of the environment variable, locking in the vendoring semantics.